modules/ztp-image-based-upgrade-talm.adoc

```diff
@@ -0,0 +1,405 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-with-talm_{context}"]
+= Upgrading the {sno} cluster with {lcao} through GitOps ZTP
```

Comment:
I find this section confusing. It's not clear to me what is pushed into Git and what is applied directly to the mgmt cluster. Also, there seem to be some overlap between these.

Since Git is supposed to reflect an *intent* while an upgrade is an *action*, I suggest we re-think this section and maybe remove it entirely for now.

Reply:
PGTs are pushed to the hub so ACM can create the policies. CGUs are applied on the hub cluster (with oc apply -f) for TALM to handle
@sakhoury Please check if my understanding is correct

Reply:
That is most certainly not correct, a PGT can _only_ exist in Git. The CustomResourceDefinition does not even exit on the Hub cluster.

But what is very confusing is the PGT which creates Policies which creates _different_ IBUs, in different stages all at once. 
Which brings me back to the question, is the IBU a singleton or not.
And what is the actual intent here?

Maybe I'm missing something, but if so, a digram of the workflow the ned user should follow would be helpful to explain the flow used when an upgrade is done using GitOps

Reply:
> pushed to the hub
Sorry, I meant to say pushed to the repo. Anyway, I'll have a discussion abou this today and clarify this asap! I'll get back to you once it's done